certified-connectors/WorkMobile/readme.md

```diff
@@ -0,0 +1,27 @@
+### WorkMobile Connector
```

Comment:
Please update readme.md file as per the template here https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
Ive filled out the readme.md file - hopefully that contains all the required parts now